ocaml/utils/language_extension.mli

```diff
@@ -49,13 +49,29 @@ with type 'a extn := 'a t
 (** Equality on language extensions *)
 val equal : 'a t -> 'b t -> bool
 
+(** The type of language extension universes. Each universe allows a set of
+    extensions, and every successive universe includes the previous one. *)
+module Universe : sig
+  type t =
+    | No_extensions
+    | Upstream_compatible
+        (** Upstream compatible extensions, also known as "erasable". *)
+    | Stable  (** Extensions of [Stable] maturity. *)
+    | Beta  (** Extensions of [Beta] maturity. *)
+    | Alpha  (** All extensions. Default. *)
+
+  val all : t list
+
+  val to_string : t -> string
+
+  val description : t -> string
+
+  val of_string : string -> t option
+end
+
 (** Disable all extensions *)
 val disable_all : unit -> unit
 
```

Comment:
It is only used in tests. I think it's worth keeping for now. Perhaps I should delete it after removing the `-extension` flag.

Reply:
I think the current design here is significantly more powerful than we need it to be. Do check in the tree (I’m on an airplane right now and can’t myself) for uses, but I’m pretty sure the only user of this interface is in some part of our ppx infrastructure, and there it simply enables all extensions so that the parser is more generous. Feel free to slash away at expressiveness here, even if that means that we lose some tests.

Reply:
That's what I want to do, but only after removing `-extension` from the CLI altogether.